canonical-webteam/snapstore-api
# Add metrics model

# Summary

- Add metrics models
- Reorder requirements.txt

# QA

Make sure tests pass (the test will check that the metrics module is well imported)
This is code moved from s.io to this module
